UnityProject/Assets/Scripts/Fire/FireSource.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Defines fire sources like candles, lighters or activated welder
+/// Fire source can lead to plasma fire or used to ignite flamable objects
+/// </summary>
+public class FireSource : MonoBehaviour, IServerSpawn
+{
+	[SerializeField]
+	[Tooltip("Does this fire emit flame from start?")]
+	private bool isBurningOnSpawn;
```

Comment:
Could you assign a value like `null` or `default` to any non-public `[SerializeField]` fields? Just helps keep the warnings down. Thanks.

Reply:
Should be done now